Flag examples in cosmology sub-package (#9835)
astropy/astropy: docs/cosmology/index.rst

you must specify a dark energy model by using one of its subclasses
instead, such as `~astropy.cosmology.FlatLambdaCDM`.

Examples
--------

..
  EXAMPLE START

>   Working with the FLRW Class Through Subclass FlatLambdaCDM

Reviewer: ```suggestion
  Working with the FlatLambdaCDM Class
```

See the `~astropy.cosmology.FLRW` and
`~astropy.cosmology.FlatLambdaCDM` object docstring for all of the
methods and attributes available.

..
  EXAMPLE END

..
  EXAMPLE START

>   Working with the FLRW Class Through Subclass LambdaCDM

Reviewer: ```suggestion
  Working with Non-flat Universes with the LambdaCDM Class
```

this more convenient, a ``clone`` operation is provided, which
allows you to make a copy with specified values changed.
Note that you cannot change the type of cosmology with this operation
(e.g., flat to non-flat).

..
  EXAMPLE START

>   Making New Cosmology Instances with the Clone Operation

Reviewer: ```suggestion
  Making New Cosmology Instances with the .clone() Method
```

  >>> WMAP9.Ode0, newcosmo.Ode0  # Indirectly changed since this is flat  # doctest: +FLOAT_CMP
  (0.7134130719051658, 0.6858130719051657)

..
  EXAMPLE END

Finding the Redshift at a Given Value of a Cosmological Quantity
----------------------------------------------------------------

If you know a cosmological quantity and you want to know the
redshift which it corresponds to, you can use ``z_at_value``.

Example
^^^^^^^

..
  EXAMPLE START

>   Finding the Redshift at a Given Value of a Cosmological Quantity

Reviewer: ```suggestion
  Compute the Redshift at a Given Universe Age
```

performance is critical, it is particularly useful to calculate them on
a grid and use interpolation.

Examples
^^^^^^^^

..
  EXAMPLE START

>   Calculating the Contribution of Photons and Neutrinos with Cosmology

Reviewer: ```suggestion
  Calculating the Contribution of Photons and Neutrinos to the Energy Density
```
